README.md

```diff
@@ -467,7 +468,70 @@ function App() {
 }
 ```
 
-### Theming
+## sx prop
+
+A special `sx` prop lets you apply styles directly to an element. When `sx` prop is specified on a component, Pigment CSS will replace any `sx` prop with `className` and `style` props at build time.
+
+The `sx` prop works on any component, including HTML elements and 3rd-party custom components as long as it is JSX.
+
+### Usage
+
+```js
+<div sx={{ display: 'flex', flexDirection: 'column' }}>
+```
+
+For a React component, it must spread the `className` and `style` to the underlying DOM element, otherwise the styles won't be applied.
+
+```js
+const AnyComponent = (props) => {
+  return <div {...props} />;
+};
+
+<AnyComponent sx={{ fontSize: 12, color: 'red' }} />;
+```
+
+> 💡 **Gotcha**: You don't need to pass `sx` prop to underlying component or DOM because it will be replaced with `className` and `style` props.
```

Comment:
What does this mean ?

Reply:
I am afraid that the user will create an intermediate component and pass the `sx` prop down, but I think it can be removed.